code/modules/mob/living/carbon/alien/diona/diona_nymph.dm

```diff
@@ -47,6 +48,11 @@
 	var/echo = 0 //if it's an echo nymph, which has unique properties
 	var/detached = FALSE
 
+/mob/living/carbon/alien/diona/get_descriptor()
+	return pick(diona_descriptors)
+
+var/list/diona_descriptors = list("rustling","skittering","friendly","leafy","frisky","sluggish","energetic","ordinary","unremarkable","sleepy","curious","bouncy","moist","lonely","sociable","chittering","beady-eyed","silent","quiet","small","tiny","big","large","slender","thick","vibrating","naive","lost","newborn","chirping","eager","shy")
```

Comment:
This is by far the worst way to preform this task. We have a entire document filled with descriptors in the config\names folder, why not just make it select one from this list, this is really really not needed and is just clutter.

You can literally do var/species_descriptors = pick(adjectives) and it will pick from the list which has a large list of usable ones that fit all mobs

Reply:
This is more copypasted conjecture than review, I'm.. not sure you actually read the adjectives file you directed me to, because using it would result in nonsense like:

-dead rat
-misty nymph
-impossible monkey
-rich mouse
-wrong golem

By contrast, my lists are catered to the specific mobs they are meant for (diseased rat, ponderous golem, etc), and I find it somewhat insulting that you consider the effort to make content tailored what it is being used for 'bloaty' and would prefer I use a text file that contains about 25% object and location based adjectives in it.